script/Ciel/Day 3/03_01_CIEL03_3_2B.txt

```diff
@@ -151,6 +151,7 @@ In my heart I still fear death,
 [sha:aefd2d0b8f749a4654f00ffc113ad7a52f422f1e]{
 -- Page 135, Offset 5192.
 -- ざあざあ、という雑音。
+// "rhythm" or "patter" would work just as well and aren't so obscure (and sus)
 A soft, steady susurration.
```

Comment:
Patter works for me, but the alliteration is too good. I'm for keeping susurration.

Reply:
alliteration is all well and good but "susurration" is a a word so terrible, the firefox spellchecker thinks its a typo. I'm for "patter"

Reply:
Obviously partial to susurration, but if it's too SAT word we can tone it down

Reply:
I'll leave it for the group PR